tests/bsim/bluetooth/tests.nrf5340bsim_nrf5340_cpunet.txt

```diff
@@ -10,7 +10,6 @@ tests/bsim/bluetooth/host/adv/extended
 tests/bsim/bluetooth/host/adv/chain
 tests/bsim/bluetooth/host/l2cap/send_on_connect
 tests/bsim/bluetooth/host/central
-tests/bsim/bluetooth/host/privacy/peripheral
```

Comment:
Please just squash this commit into the commit in which you just added it ("tests: bsim: bluetooth: host: enable on nrf5340bsim/nrf5340/cpunet")